### ISSUE TITLE
Agile Fighter, Alien Fighter in Battle of 1996(Independent Day)

### ISSUE THREAD

**Issue by 1103CA** [PygmalionOfCyprus/cmo-db-requests#4054]
### DB Selector

Both

### Hypothetical

Yes

### Name

Agile Fighter (Harvester Attacker)

### Country

Unknown

### In Commission

1996

### Length (m)

19.5

### Wingspan (m)

20

### Height (m)

4.75

### Empty Weight (kg)

20000

### Maximum Takeoff Weight (MTOW) (kg)

20400

### Payload Weight (kg)

450

### Crew

2

### Propulsion

1x Generic Anti-gravity systems

### Rate of Climb (ft/min)

2800ft/min

### Service Ceiling (ft)

60000ft

### Military/Afterburner Speed (kt)

700ft

### Cruise Speed (kt)

480ft

### Supercruise

Yes

### Ferry Range (nm)

2500nm

### Endurance (min)

2400min

### Takeoff / Landing Distance (m)

0m(VTOL)

### Fuel Capacity (kg)

0

### Sensors

1x Mk1 Eyeball
1x Generic Alien Lidar
1x Generic Alien OECM

### Mounts

2x Generic Alien Ion Cannon(Always Installed)

### Loadout Options

Alien Generic Mission
2x Generic Alien Ion Cannon

### Comms/Datalinks
1x Generic Alien Brainwave System

_No response_

### Aircraft Codes

Supermanouverability, Terrain Avoidance, Terrain Following, Night Navigation (Ferry, Air-to-Air, Air-to-Surface Missiles), Bombsight - Advanced Navigation (INS/GPS), Helmet Mounted Sight / Display (HMS/HMD)

### Comments

The Attacker, or officially known as Agile Fighter, is a Harvester fighter craft. Attackers are deployed by City Destroyers, as well as Motherships.
The Attacker is a very versatile aircraft. Though their method of propulsion was never elaborated on, it is strongly evident that Attackers have transcended the need for traditional liquid or solid-fuel combustion engines at least while hovering in place, given their lack of the incredible heat and exhaust output necessary to support something its apparent size and mass, such as from a jet engine.

Despite not having wings or maneuvering flaps for stability, the Attacker is capable of stable atmospheric flight even at speeds reaching or exceeding the sound barrier while still maintaining an incredible degree of maneuverability and have been demonstrated to have fantastic acceleration power, going from a completely stationary hover to presumably several hundred miles per hour in just a few seconds. It should be noted that all of these handling characteristics have been observed from a badly damaged crash specimen recovered and then repaired from the Roswell incident of 1947; it is probable that a mechanically undamaged and fully stocked Attacker may exhibit even better handling characteristics.

![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/151608074/18a0864c-9d13-4854-92ba-ae47c894994e)
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/151608074/83135ecc-31d1-455a-848a-47607b8e733d)
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/151608074/0b8296d3-da73-4386-a2c5-5de42717b8f4)

### Sources

https://independenceday.fandom.com/wiki/Attacker
https://www.liveauctioneers.com/item/166788073_independence-day-alien-attacker-agile-fighter-filming-miniature-with-dvd
https://www.youtube.com/watch?v=vObiMAPTjTU
https://independence-day.fandom.com/wiki/Harvester_Attacker

**Comment by bqz2801** [comment]
yes please, this would be just awesome. whether for shits and giggles or in all seriousness

**Comment by TTZ32** [comment]
100% agree with this. Ive made make shift alien scenarios using SR72 with laser , EMP and every oecm/decm available but this would be much better.

**Comment by KMPDB** [comment]
Request it in another simulation.
